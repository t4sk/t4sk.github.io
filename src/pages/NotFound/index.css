.NotFound {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  padding-top: 50px;
  padding-bottom: 50px;
}
.NotFound-header {
  font-weight: 700;
  font-size: 6rem;
  color: #20232a;
}
.NotFound-sub-header {
  font-size: 2rem;
  color: #aaa;
}
