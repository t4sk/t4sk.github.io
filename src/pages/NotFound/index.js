import React from "react"
import PropTypes from "prop-types"
import PageLayout from "../../components/PageLayout"
import SEO from "../../components/SEO"
import styles from "./index.module.css"

function NotFound(props) {
  return (
    <PageLayout>
      <SEO title="Not Found | Smart Contract Programmer" />
      <div className={styles.NotFound}>
        <h1 className={styles.header}>404</h1>
        <div className={styles.subHeader}>Not Found</div>
      </div>
    </PageLayout>
  )
}

NotFound.propTypes = {}

export default NotFound
