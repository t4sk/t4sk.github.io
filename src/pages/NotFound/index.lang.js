const LANG = {
  en: {
    "Not Found": "Not Found",
  },
  jp: {
    "Not Found": "ページが見つかりません",
  },
}
export default LANG
