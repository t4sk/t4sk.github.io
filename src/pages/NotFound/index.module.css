.NotFound {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  padding-top: 50px;
  padding-bottom: 50px;
}
.header {
  font-weight: 700;
  font-size: 6rem;
  color: #20232a;
}
.subHeader {
  font-size: 2rem;
  color: #aaa;
}
