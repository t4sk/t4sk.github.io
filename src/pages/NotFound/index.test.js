import React from "react"
import { shallow } from "enzyme"
import NotFound from "./index"

const props = {
  location: {
    pathname: "/en",
  },
}

test("en", () => {
  const component = shallow(
    <NotFound
      {...props}
      location={{
        pathname: "/en/not/found",
      }}
    />
  )

  expect(component).toMatchSnapshot()
})

test("jp", () => {
  const component = shallow(
    <NotFound
      {...props}
      location={{
        pathname: "/jp/not/found",
      }}
    />
  )

  expect(component).toMatchSnapshot()
})

test("not en nor jp", () => {
  const component = shallow(
    <NotFound
      {...props}
      location={{
        pathname: "/not/found",
      }}
    />
  )

  expect(component).toMatchSnapshot()
})
