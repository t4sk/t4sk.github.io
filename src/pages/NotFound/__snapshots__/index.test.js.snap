// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<PageLayout>
  <SEO
    description="Learn how to program smart contracts."
    keywords={
      Array [
        "smart contract",
        "solidity",
        "ethereum",
      ]
    }
    lang="en"
    meta={Array []}
    title="Not Found | Smart Contract Programmer"
  />
  <div
    className="NotFound"
  >
    <h1
      className="header"
    >
      404
    </h1>
    <div
      className="subHeader"
    >
      Not Found
    </div>
  </div>
</PageLayout>
`;
